umap/settings/base.py

```diff
@@ -114,6 +114,9 @@
 )
 DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
 
+EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+FROM_EMAIL = None
```

Comment:
I still wonder if we should add `FROM_EMAIL = "test@example.com"` to `local.py.sample` to see the feature in action when developing or if that's really an on-demand one 🤔